src/zoid/buttons/util.js

```diff
@@ -306,6 +307,7 @@ export function applePaySession(): ?ApplePaySessionConfigRequest {
 export function getButtonExperiments(): EligibilityExperiment {
   return {
     ...getVenmoExperiment(),
+    ...getFirstRenderExperiments(),
```

Comment:
Please validate if there are any other methods using `getButtonExperiments()` and may have any side affects to due this change.

Reply:
That I did 😄  Its only. used in zoid/buttons/component.jsx 🚀